libs/community/langchain_community/chat_loaders/facebook_messenger.py

```diff
@@ -37,7 +37,13 @@ def lazy_load(self) -> Iterator[ChatSession]:
             data = json.load(f)
         sorted_data = sorted(data["messages"], key=lambda x: x["timestamp_ms"])
         messages = []
-        for m in sorted_data:
+        for index, m in enumerate(sorted_data):
+            if "content" not in m:
+                print(
+                    f"""Skipping Message No.
+                    {index+1} as no content is present in the message"""
+                )
```

Comment:
Yeap but elsewhere they are also using print as well. See other tools

Reply:
Can you provide some tools that include these print statements?

IMO, I think that the user wouldn't want to verbose this since it is expected to be a lot of non-content messages in his chat.

Additionally, I think that the statement is quite unintuitive, especially if we use FolderFacebookMessengerChatLoader (which calls SingleFileFacebookMessengerChatLoader). In this case, it will print the index without providing any information related to the current file. This is kind of hard to debug imo.

Reply:
@th4tkh13m check this out: https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/utilities/google_lens.py

Reply:
I do agree that messages might be too verbose for the user but still we need to inform the user that not all of the messages are being loaded otherwise their might be confusion.

Reply:
we should usually use logger, the print statements are likely non-intentional or a special case

Reply:
By the way, I think that we should modify the log statement a little bit. Something like:
```python
logger.info(
                    f"""{file_name}: Skipping Message No.
                    {index+1} as no content is present in the message"""
                )
```

should be more informative. It provides the file name and the index (we can change this to the key, which is the timestamp of the message) so that we can track down in what message file and what messages are missing content.